### PR TITLE
runtime-rs: update load_config comment

### PR DESCRIPTION
**Pull request by studychao**
Since shimv2 create task option is already implemented, we need to update the corresponding comments.

Also, the ordering is also updated to fit with the code.

fixes: #3961

### CODE REVIEW

File: src/runtime-rs/crates/runtimes/src/manager.rs [hunk 1]
```diff
@@ -353,9 +353,8 @@ impl RuntimeHandlerManager {
 
 /// Config override ordering(high to low):
 /// 1. podsandbox annotation
-/// 2. shimv2 create task option
-/// TODO: https://github.com/kata-containers/kata-containers/issues/3961
-/// 3. environment
+/// 2. environment variable
+/// 3. shimv2 create task option
```
Comment (fidencio) [inline src/runtime-rs/crates/runtimes/src/manager.rs:357]: Please, add `/// 4. Default locations, based on https://github.com/kata-containers/kata-containers/blob/dc42f0a33b4e60750cda9aba241d25a88cbc65c2/src/libs/kata-types/src/config/default.rs#L14-L20

Comment (studychao) [reply src/runtime-rs/crates/runtimes/src/manager.rs:357]: Thanks, changed.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @studychao!

File: src/runtime-rs/crates/runtimes/src/manager.rs [hunk 2]
```diff
@@ -353,9 +353,10 @@ impl RuntimeHandlerManager {
...
+/// 4. If above three are not set, then default path in kata-containers/src/libs/kata-types/src/config/default.rs
```
Comment (c3d) [inline src/runtime-rs/crates/runtimes/src/manager.rs:358]:
```suggestion
/// 4. If above three are not set, then get default path from DEFAULT_RUNTIME_CONFIGURATIONS
///    in kata-containers/src/libs/kata-types/src/config/default.rs, in array order
```

**Review by c3d — APPROVED** [review]
Looks good to me, but I suggested a slight wording change to indicate that `DEFAULT_RUNTIME_CONFIGURATIONS` itself has an order, since it's an array.

**Review by quanweiZhou — APPROVED** [review]

File: src/runtime-rs/crates/runtimes/src/manager.rs (hunk 2 above)
Comment (studychao) [reply src/runtime-rs/crates/runtimes/src/manager.rs]: Thanks, changed.

**Pull request merged**